modules/core/src/views/view.ts

```diff
@@ -31,8 +31,12 @@ export type CommonViewProps<ViewState> = {
     top?: number | string;
     bottom?: number | string;
   } | null;
-  /** When using multiple views, set this flag to wipe the pixels drawn by other overlaping views */
+  /** When using multiple views, set this flag to wipe the pixels drawn by other overlapping views. Default `undefined` */
   clear?: boolean;
+  /** Color to clear the viewport with, in RGBA format [r, g, b, a]. Values are 0-255. Default `undefined` */
+  clearColor?: number[];
+  /** Whether to clear the depth buffer. Default `undefined` */
+  clearDepth?: boolean;
```

Comment:
As the docs note, deck clears the entire screen before each render so I assumed these are per-view overrides, and therefore, should all be undefined by default (e.g. don't perform clearing)

Reply:
I've updated this PR and think the implementation is aligned with your understanding now.

There might be some more discussion to have around how we're setting up render passes. We set up [one main pass](https://github.com/visgl/deck.gl/blob/e25d735d2d6d8b611a65e60b3b934ab407d006cd/modules/core/src/passes/layers-pass.ts#L71-L92) prior to inspecting the views, and then [more brief passes](https://github.com/visgl/deck.gl/blob/e25d735d2d6d8b611a65e60b3b934ab407d006cd/modules/core/src/passes/layers-pass.ts#L231-L240) for each viewport with clearing enabled.

I'm wondering why we don't just set up a new render pass for each view? I'm assuming there was some good reason to do it as we are currently, or maybe this is left over and a candidate for a refactor?